docs/fa/class-dfa.md

```diff
@@ -455,4 +455,5 @@ dfa.show_diagram(path='./dfa1.png')
 
 [Next: NFA Class](class-nfa.md)  
 [FA Class](class-fa.md)  
+[Examples](examples.md)  
```

Comment:
@eliotwrobson I am hesitant to add Examples to the bottom navigation of each docs page. I see the bottom navigation for one of two things:

1. Next section
2. Page hierarchy

The Examples section doesn't fall into either of those two categories (it's generally a non-adjacent sibling page). Plus, it might imply that the examples are all DFA-specific, whereas it actually encompasses everything for FA.

That's my view anyway. But hey, I'm open to hearing your thoughts on this!

Reply:
Ah, I think that's perfectly sound reasoning. TBH I didn't really have an opinion, so I kinda just threw the links to the examples at the bottom navigation. Will remove a little later.

Reply:
Done here https://github.com/caleb531/automata/pull/146/commits/2ce6816e437d8ca17b4e095ab4922e18eb4421f7